static_src/components/users_invite.jsx

```diff
@@ -93,6 +119,14 @@ export default class UsersInvite extends React.Component {
     } else {
       content = '';
     }
+    if (this.props.notice) {
```

Comment:
currently this notice only comes up during the invite process. you might want to move this up to the conditional where we only display this invite notice upon have the right access.